src/requisition-search/requisition-search.controller.js

```diff
@@ -239,10 +268,26 @@
             stateParams.initiatedDateFrom = vm.startDate ? $filter('isoDate')(vm.startDate) : null;
             stateParams.initiatedDateTo = vm.endDate ? $filter('isoDate')(vm.endDate) : null;
             stateParams.offline = vm.offline;
+            stateParams.requisitionStatus = vm.selectedStatus;
 
             $state.go('openlmis.requisitions.search', stateParams, {
                 reload: true
             });
         }
+
+        /**
+         * @ngdoc method
+         * @methodOf requisition-search.controller:RequisitionViewController
+         * @name getStatusDisplay
+         *
+         * @description
+         * Evaluates display name for given status.
+         * 
+         * @param   {String} status given status
+         * @returns {String}        translated status name
+         */
+        function getStatusDisplay(status) {
```

Comment:
Can't we use the filter directly in the html?

Reply:
Done